README.md

```diff
@@ -50,6 +50,40 @@ The player "○" wins the game since he placed his marks in a row vertically (B2
 
 The player "○" wins the game since he placed his marks in a row diagonally (A3-B2-B1).
 
+## Code Formatting Check
```

Comment:
Drop the section "Code Formatting Check."

Instead, I would place contributing guide like https://github.com/sysprog21/rv32emu/blob/master/CONTRIBUTING.md .